lib/spack/llnl/util/cpu/microarchitectures.json

```
@@ -1239,17 +1252,30 @@
           },
           {
             "versions": "7:7.9",
-            "flags": "-arch=armv8.2a+crc+crypt+fp16"
+            "flags": "-arch=armv8.2a+crc+crypto+fp16"
           },
           {
             "versions": "8:",
-            "flags": "-arch=armv8.2a+crc+aes+sh2+fp16+sve -msve-vector-bits=512"
+            "flags": "-arch=armv8.2a+crc+aes+sha2+fp16+sve -msve-vector-bits=512"
```

Fixes `-arch` to `-march` is deleted. Please fix typo again.
```suggestion
            "flags": "-march=armv8.2a+crc+aes+sha2+fp16+sve -msve-vector-bits=512"
```

---

Apologies, I didn't pull before rebasing.